### ISSUE TITLE
Issue with "Avoid expression statements that implicitly ignore value" code style fixer

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#61467]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Issue-with-Avoid-expression-statements-/10049283)._

---
The **Avoid expression statements that implicitly ignore value** code style setting can be set to _Unused_ local or _Discard_:

![Immagine 2022-05-20 224939.png](https://aka.ms/dc/image?name=B697dfe25c2c64fb4b9da4f385167d121637887262102681843_Immagine_2022-05-20_224939.png&tid=697dfe25c2c64fb4b9da4f385167d121637887262102681843)

However, I think that these settings do not cover all the scenarios, because there are cases in which we simply want to ignore the return values. I have attached a recording to show such a situation, that is also available at [https://www.youtube.com/watch?v=ZuSf84-Mw44](https://www.youtube.com/watch?v=ZuSf84-Mw44).

---
#### Feedback Bot on 5/22/2022, 07:54 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by sharwell** [comment]
It appears that the Code Cleanup operation is applying code fixes for diagnostics that are marked as Refactoring Only. This should not occur.

**Comment by kirsan31** [comment]
I think `dotnet_diagnostic.IDE0058.severity = none` in `.editorconfig` will solve the problem?

**Comment by marcominerva** [comment]
Yes, if I use an `.editorconfig` file, this setting solves the problem. But, if I don't use an `.editorconfig`, the issue remains.

**Comment by sybren9** [comment]
There's 2 separate issues here at play which both apply to the same problem;

1. Code Cleanup operation is applying code fixes for diagnostics that are marked as Refactoring Only, which should not happen as pointed out by @sharwell 
2. Visual Studio is missing severity 'none' as an option in the code style menu.

2nd issue seems to also have been mentioned a bunch of times ( [here](https://github.com/dotnet/roslyn/issues/56461), [here](https://stackoverflow.com/questions/63997189/visual-studio-code-style-preferences-severity-of-none-is-absent)

Seems the answer for the 2nd issue has always been 'generate.editorconfig files and manually edit them' which makes me question as well what the point of the Code Syle menu really is.


Either way issue 1 needs to be looked at asap, makes code cleanup quite unusable in this state
